src/Validator/SDLValidationContext.php

```diff
@@ -2,6 +2,42 @@
 
 namespace GraphQL\Validator;
 
-class SDLValidationContext extends ASTValidationContext
+use GraphQL\Error\Error;
+use GraphQL\Language\AST\DocumentNode;
+use GraphQL\Type\Schema;
+
+class SDLValidationContext implements ValidationContext
 {
+    protected DocumentNode $ast;
+
+    protected ?Schema $schema;
+
+    /** @var array<int, Error> */
```

Comment:
list?

Reply:
I would do one full pass over all instances of `array<int, ` and check if they can be `list<` in a separate PR - see https://github.com/webonyx/graphql-php/issues/811